app/models/profile.server.ts

```diff
@@ -30,6 +30,31 @@ export async function createProfile(data: Prisma.ProfilesCreateInput) {
   return prisma.profiles.create({ data });
 }
 
+export async function consolidateProfilesByEmail(data: Prisma.ProfilesCreateInput[], db: PrismaClient) {
+  const profileMails = data.map((profile) => {
+    return profile.email
+  })
+  console.info(`Starting upsert profiles to DB`);
+
+  const profilesUpsert = data.map((profile) => {
+    return db.profiles.upsert({
+      where: { email: profile.email },
+      update: {
+        ...profile,
+      },
+      create: {
+        ...profile,
+      },
+    })
+  });
+
+  console.info(`Starting delete profiles from DB`);
+  await db.$transaction([
+    db.$queryRaw`UPDATE "Profiles" SET deleted=TRUE WHERE email NOT IN (${profileMails.join(",")})`,
```

Comment:
Do we want to keep this `deleted` column? We probably got it from WizelineOS, but we are now using the datalake. We could instead set `status` to `Terminated` instead, and remove the `deleted` column.

What do you think?

Reply:
For now i don't recall if we are using this column in any other part of the code, so we could get rid of it but i will have to check if any component or function is expecting this to exist and that it would not fail

Reply:
I made the change on my last commit to this branch:

> feat: remove deleted and terminatedAt columns that are not synced, replace status with employeeStatus and benchStatus, add businessUnit on profiles